### PR TITLE
Added simple LayerCarousel

### PR DESCRIPTION
**Pull request by weskamm**
This adds a simple Layer Carousel with preview functionality, either a static image or a dynamic GetMap request

### CODE REVIEW

**Comment by KaiVolland** [comment]
Could you add a Screenshot?

File: packages/baseclient-components/src/component/LayerCarousel/LayerCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        <br/>
+        <img
+          src={layer.get('imageUrl') ? layer.get('imageUrl') : this.getLayerPreview(layer)}
+          height={128}
+        />
+      </div>
+    );
+
+    return (
+      <div className='carousel-wrapper'>
```
Comment (KaiVolland) [inline packages/baseclient-components/src/component/LayerCarousel/LayerCarousel.tsx]: Wrapping div is unneeded just add `className` directly to the `Carousel`.

**Comment by KaiVolland** [comment]
Very nice but this should be a `react-geo` component.

File: packages/baseclient-components/src/component/LayerCarousel/LayerCarousel.tsx (hunk 1 above)
Comment (weskamm) [reply packages/baseclient-components/src/component/LayerCarousel/LayerCarousel.tsx]: changing this totally breaks the carousel, so it will stay for now

**Comment by weskamm** [comment]
You are right, i am currently missing some time to swap it over

**Pull request merged**